cypress/integration/Login/login.spec.js

```diff
@@ -0,0 +1,50 @@
+import LoginPage from "../../Pages/Login";
+
+describe("Login Page", () => {
+  beforeEach(() => {
+    LoginPage.open();
```

Comment:
In other tests, we would often login first in a before(() => {... }) - you can add a global login function in the commands.js file later on so you can do cy.login(email, password); and then in the beforeEach you would do things like BounceRulesPage.open();